packages/dd-trace/src/appsec/iast/taint-tracking/operations-taint-object.js

```diff
@@ -25,6 +25,7 @@ function taintObject (iastContext, object, type) {
           } else {
             parent[key] = tainted
           }
+          // eslint-disable-next-line @datadog/safe-typeof-object
```

Comment:
Why not just add the check for `null` instead of disabling the rule? Same question for pretty much all other occurrences.